### PR TITLE
Edit to findoutlie/detectors.py

### PR DESCRIPTION
**Pull request by Yingjuan**
For week 6 session 1 homework

### CODE REVIEW

File: findoutlie/detectors.py [hunk 1]
```diff
@@ -51,3 +51,15 @@ def iqr_detector(measures, iqr_proportion=1.5):
     # * You'll likely need np.logical_or
     # https://textbook.nipraxis.org/numpy_logical.html
     # +++your code here+++
+
+    Q1 = np.percentile(measures, 25)
+    Q3 = np.percentile(measures, 75)
```
Comment (matthew-brett) [inline findoutlie/detectors.py]: Or:

```suggestion
    Q1, Q3 = np.percentile(measures, [25, 75])
```

File: findoutlie/detectors.py [hunk 2]
```diff
@@ -51,3 +51,15 @@ def iqr_detector(measures, iqr_proportion=1.5):
...
+    IQR = Q3 - Q1
+
+    upper_boundary = Q3 + IQR * iqr_proportion
+    lower_boundary = Q1 - IQR * iqr_proportion
+
+    outliers = np.logical_or(measures < lower_boundary, measures > upper_boundary)
```
Comment (matthew-brett) [inline findoutlie/detectors.py]: Or:

```suggestion
    outliers = (measures < lower_boundary ) | (measures > upper_boundary)
```

**Review by matthew-brett — COMMENTED** [review]
Just two suggestions for style - generally good.

File: findoutlie/detectors.py [hunk 3]
```diff
@@ -51,3 +51,14 @@ def iqr_detector(measures, iqr_proportion=1.5):
...
     # https://textbook.nipraxis.org/numpy_logical.html
     # +++your code here+++
+
+    Q1, Q3 = np.percentile(measures, [25, 75])
+    IQR = Q3 - Q1
+
+    upper_boundary = Q3 + IQR * iqr_proportion
+    lower_boundary = Q1 - IQR * iqr_proportion
+
+    outliers = (measures < lower_boundary | (measures > upper_boundary)
```
Comment (Yingjuan) [inline findoutlie/detectors.py:61]: outliers = (measures < lower_boundary ) | (measures > upper_boundary)

**Pull request merged**